items.json

```diff
@@ -77922,10 +77922,10 @@
     "name": "Long Thamaskene Tipped Spear +1",
     "culture": "Aserai",
     "type": "Polearm",
-    "price": 9497,
-    "weight": 2.07,
+    "price": 10965,
+    "weight": 1.96,
     "heirloomLevel": 1,
-    "tier": 8.046889,
+    "tier": 8.72598,
     "requirement": 0,

```

Comment:
opriginal tier was 9.74 , it's a bit too weak. Try not to get heirloomed version further than 1 tier. 0.8/0.7 is acceptable